Makefile.ci.toml

```diff
@@ -0,0 +1,27 @@
+# Format
+[tasks.cargo-fmt]
+category = "LOCAL"
+command = "cargo"
+condition = { channels = ["nightly"] }
+args = ["fmt", "--all", "--check"]
+
+# CI
+[tasks.ci-format]
+category = "CI - CHECK"
+dependencies = ["cargo-fmt"]
+
+[tasks.ci-clippy]
+category = "CI - CHECK"
+command = "cargo"
+args = [
+    "clippy",
+    "--all-targets",
+    "--all-features",
+    "--tests",
+    "--benches",
+    "--examples",
+    "--bins",
+    "--",
```

Comment:
I don't see the `--workspace` switch. Which crates does the command apply to in this case? Wondering if this covers everything we want.

Even with the `--workspace` switch, I don't think it would run on the pessimistic proof crates as these technically are under a separate workspace. Or am I mistaken here and it processes all the local dependencies anyway?